dev/docker-compose.yaml

```
@@ -43,13 +43,18 @@ services:
       - db
     command: >
       sh -c "
-            echo '**** Checking WordPress Plugins, Core and Site List ***' &&
-            wp plugin update --all --minor  &&
-            wp core check-update &&
-            wp site list --fields=url &&
-            echo '*** Backing up Database in case of disaster ***' &&
-            wp db export /tmp/WordPress/all-sites-$(date '+%Y-%m-%d').sql --add-drop-table &&
-            echo '*** The WordPress CLI is now available in any terminal window `wp help` ***' &&
+            echo '**** Checking WordPress Plugins, Core and Site List ***' \
+            if ! wp core is-installed 2&> /dev/null; then \
+              # WP is not installed. Let's try installing it. \
+              echo '** installing wordpress core' \
+              wp core multisite-install --url=localhost --title='Local Development' --admin_user=admin --admin_password=password --admin_email=admin@example.com \
+            else
+              wp core check-update \
+              wp plugin update --all --minor  \
+              echo '*** Backing up Database in case of disaster ***' \
+              wp db export /tmp/WordPress/all-sites-$(date '+%Y-%m-%d-%H:%M').sql --add-drop-table \
+            fi \
+            echo '*** The WordPress CLI is now available in any terminal window `wp help` ***' \
```

```suggestion
            if ! wp core is-installed 2&> /dev/null; then \
              # WP is not installed. Let's try installing it. \
              echo '** Installing WordPress core' \
              wp core multisite-install --url=localhost --title='Local Development' --admin_user=admin --admin_password=password --admin_email=admin@example.com \
            else
              echo '**** Checking WordPress Plugins, Core and Site List ***' \
              wp core check-update \
              wp plugin update --all --minor  \
              echo '*** Backing up Database in case of disaster ***' \
              wp db export /tmp/WordPress/all-sites-$(date '+%Y-%m-%d-%H:%M').sql --add-drop-table \
            fi \
            echo '*** The WordPress CLI is now available in any terminal window `wp help` ***' \
```
Does this still work if our MULTISITE env variable is 0? It looks like it will install it as multisite but then if the WP config isn't set to multisite it would fail.

---

maybe, everyone should be on multi-site, what i know is i didn't have to set it to 0 first and then 1 , i did find that i had to run the wp_start 2x for it to work from a fresh database, i suspect the first time it creates the db and sets it up, but 2x it runs it fine, and all i have to do is import the database via the wp-cli

---

We could maybe remove the `MULTISITE` variable from our .env and hardcode it to true then if this script can handle the setup for us.